packages/gatsby-theme-carbon/src/components/FeatureCard/feature-card.scss

```diff
@@ -33,6 +34,14 @@
   background: $carbon--gray-80; //$hover-ui for gray 90 theme
 }
 
+.#{$prefix}--feature-card__link:focus {
+  outline: none;
+
+  .gatsby-resp-image-wrapper {
+    @include focus-outline;
```

Comment:
Please note that this ends up adding a 1px outline.
But please let me know if you'd like a thicker outline, like 2px. That would be a custom rule outside the mixin (because offset won't work here), but I can switch that easily 👍

Reply:
@jendowns If you could increase it to match the other components that'd be great. We've been using this rather than the focus-outline helper:

```
  outline: 2px solid $focus;
  outline-offset: -2px;
```